cluster/gce/util.sh

```diff
@@ -2342,6 +2350,8 @@ function create-subnetworks() {
 
   # Look for the alias subnet, it must exist and have a secondary
   # range configured.
+  # Deliberately declare and assign at the same time or the CI pipeline fails
```

Comment:
huh? are we depending on masking this error?

Reply:
I just couldn't wrap my head around it. I fail to see why it does not work having the declaration and assignment separated, but repeated experiments with the CI pipeline suggests reality is something else. :-)